docs/ide-usage.md

```diff
@@ -85,104 +85,63 @@ The following plugins are needed:
 
 ## IntelliJ IDEA
 
+> - Replace all references of `IntelliJ IDEA → Preferences` to `File → Settings` if you are using Windows or Linux.
+
 Supported IntelliJ versions: IntelliJ IDEA Ultimate Edition (required to work with Google App Engine).
 You can sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a student registered in an educational institution.
 
-**NOTE**
-> If you are migrating from Eclipse, you should delete `.project`, `.classpath`, `.launches`,
-> `src/main/webapp/WEB-INF/classes`, `src/main/webapp/WEB-INF/lib` and `src/main/webapp/WEB-INF/appengine-generated`.\
-> You can also copy `local_db.bin` from `src/main/webapp/WEB-INF/appengine-generated` to the location your dev server in
-> IntelliJ reads the datastore from after your IntelliJ setup. See [dev server setup](development.md#with-intellij) for
-> more details.
-
 ### Prerequisites
-1. You need a Java 7 SDK with the name `1.7` defined in IntelliJ IDEA as follows:
-
-    1. If you have no current projects open, click `Configure → Project Defaults → Project Structure`.\
-       **OR**\
-       If you currently have projects open, click `File → Project Structure`.
-    1. Select SDKs in Platform Settings and check if there is a SDK named `1.7` with a JDK home path pointing to a
-       JDK 7 path. Otherwise add a new SDK using JDK 7 with a name of `1.7`.
-
-1. You need the system property `ide=idea` for `Gradle VM options:` under `Global Gradle settings` as follows:
-
-    1. If you have no current projects open, click `Configure → Settings/Preferences`.\
-       **OR**\
-       If you currently have projects open, click `File → Settings` or `IntelliJ IDEA → Preferences`.
-    1. Go to `Build, Execution, Deployment → Build Tools → Gradle`. Under `Global Gradle settings`,
-       add `-Dide=idea` to `Gradle VM options:`.
-
-1. You need to have an application server named `AppEngine Dev <version>` pointing to the SDK you downloaded previously as follows: 
-    
-    1. If you have no current projects open, click `Configure → Settings/Preferences`.\
-       **OR**\
-       If you currently have projects open, click `File → Settings` or `IntelliJ IDEA → Preferences`.
-    1. Go to `Build, Execution, Deployment → Application Servers`.
-    1. Click `+ → Google App Engine Dev Server → ... `. Select the App Engine SDK (`appengine-java-sdk-<version>` sub-folder)
-       you downloaded in Step 3 of the [Setting up a development environment](setting-up.md) guide.
-    1. Ensure the `Name:` is of `AppEngine Dev <version>`.
-
-### Automated Setup
+1. You need a Java 8 SDK with the name `1.8` defined in IntelliJ IDEA as follows:
 
-If you do not wish to use the automated setup, you can follow the [manual setup](#manual-setup) below.
+   * Click `Configure → Project Defaults → Project Structure` (or `File → Project Structure` if a project is currently open).
```

Comment:
Actually, why not just ask everyone to close any open project they're in when setting up? They eventually have to close their open project when switching to TEAMMATES, right?

Reply:
The old instructions were like that. @LiHaoTan remind us of the reason you changed it?

Reply:
@whipermr5 I'm not sure what you mean by me changing it though?

Reply:
Yup! https://github.com/TEAMMATES/teammates/pull/7354/files#diff-6c717a8c6a9d2d4e6b3b0779053fec18L89

Reply:
Part of the reason was to avoid setting defaults, but because the instructions got split into sections that are not so clear.

Of course, one could say that for this specific section we can ask them to close the project, but forcing the user to close the project just to set up this step feels intrusive to me, for the reasons I mentioned above here: https://github.com/TEAMMATES/teammates/pull/8096#discussion_r144700331